### PR TITLE
made scholarship select course page

### PR DESCRIPTION
**Pull request by TheFeesh**

### CODE REVIEW

File: src/pages/scholarshipSelectCourse/scholarshipSelectCourse.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import { useState, useEffect } from "react";
+import { useNavigate, Link } from "react-router-dom";
+import { useFlashMsg } from "../../services/flashMsg";
+import sendReq from "../../services/sendReq";
+import validateUuid from "../../validation/uuid";
+import baseUrl from '../../apiUrls';
+import Button from "../../components/button/button";
+import Page from "../../components/page/page";
+import 'react-phone-input-2/lib/style.css';
```
Comment (AlexanderZhou39) [inline src/pages/scholarshipSelectCourse/scholarshipSelectCourse.js:9]: this import is useless

**Review by AlexanderZhou39 — CHANGES_REQUESTED** [review]
include a screenshot as well

**Comment by TheFeesh** [comment]
image

File: src/pages/scholarshipSelectCourse/scholarshipSelectCourse.js (hunk 1 above)
Comment (TheFeesh) [reply src/pages/scholarshipSelectCourse/scholarshipSelectCourse.js:9]: wait 6-8 seems to be used?

File: src/pages/scholarshipSelectCourse/scholarshipSelectCourse.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	const listItems = courseList.map((course) =>
+		<option value={course.id} key={course.name.toString()} >{course.name}</option>
+	);
+
+	const onSubmit = () => {
+		const [err] = validateUuid(courseId);
+		if (err) {
+			setErrorMessage('Please choose a course');
+		} else {
+			navigate(`/courses/${courseId}/batches`);
```
Comment (AlexanderZhou39) [inline src/pages/scholarshipSelectCourse/scholarshipSelectCourse.js:32]: remove line 32 as well

**Review by AlexanderZhou39 — APPROVED** [review]

**Pull request merged**